enhancement: updates README files to point to microsite (#156)
hashicorp/flight: README.md

# Flight Icons

[![Project Status: WIP – Initial development is in progress, but there has not yet been a stable, usable release suitable for the public.](https://www.repostatus.org/badges/latest/wip.svg)](https://www.repostatus.org/#wip)

[@hashicorp/ember-flight-icons](https://github.com/hashicorp/flight/blob/main/ember-flight-icons/)

[![npm version](https://badge.fury.io/js/%40hashicorp%2Fember-flight-icons.svg)](https://badge.fury.io/js/%40hashicorp%2Fember-flight-icons)

@hashicorp/flight-icons

[![npm version](https://badge.fury.io/js/%40hashicorp%2Fflight-icons.svg)](https://badge.fury.io/js/%40hashicorp%2Fflight-icons)

## Installation and Usage Instructions

  The micro-site for this project is [https://flight-staging.vercel.app/](https://flight-staging.vercel.app/)

> Instructions for installation and use are available, as well as a searchable list of available icons and other useful information for designers and developers.

Reviewer: ```suggestion
Instructions for installation and use are available, as well as a searchable list of available icons and the documentation for designers and developers.
```
